packages/marketplace/contracts/exchange/Exchange.sol

```diff
@@ -27,27 +31,61 @@ contract Exchange is ExchangeCore, TransferManager, ERC2771HandlerUpgradeable {
         uint256 newProtocolFeeSecondary,
         address newDefaultFeeReceiver,
         IRoyaltiesProvider newRoyaltiesProvider,
-        address orderValidatorAdress,
+        IOrderValidator orderValidatorAddress,
         bool newNativeOrder,
         bool newMetaNative
     ) external initializer {
         __ERC2771Handler_init(newTrustedForwarder);
-        __Ownable_init();
+        // TODO: Switch to a version that takes an admin address
+        __Ownable_init_unchained();
```

Comment:
Can we call `__Ownable_init()` as a matter of best practice, rather than the `__Ownable_init_unchained()`? See https://forum.openzeppelin.com/t/why-keep-ownable-init-beside-ownable-init-unchained/25579